documents/integrate-with-hotwax/api/carriers/hotwax-carrier-api.md

```diff
@@ -0,0 +1,195 @@
+# HotWax Carrier Integration API Documentation
+
+## Overview
+
+This document provides information on how to integrate a carrier with the HotWax platform. The integration involves sending a JSON request containing shipment details and receiving a JSON response with shipping label information.
+
+## Endpoint
+
+**URL**: `<insert_endpoint_here>`  
```

Comment:
Which URL is this supposed to be? Is this the endpoint of the carrier which is receiving this JSON? @AjinkyaM1

Reply:
Removed the endpoint from this document. Initially, I planned to include the carrier's endpoint, but since third parties can integrate with us, the endpoints may vary. Additionally, the endpoints can differ for each carrier.

Reply:
Also added the document with the rate and void label requests.